packages/sdk/app-framework/package.json

```diff
@@ -36,10 +36,14 @@
     "zod": "^3.22.4"
   },
   "devDependencies": {
+    "@dxos/react-ui": "workspace:*",
+    "@dxos/react-ui-theme": "workspace:*",
     "@types/react": "~18.2.0",
     "react": "~18.2.0"
   },
   "peerDependencies": {
+    "@dxos/react-ui": "workspace:*",
+    "@dxos/react-ui-theme": "workspace:*",
```

Comment:
I've specifically been trying to avoid introducing these dependencies here. App framework shouldn't take any opinion on your UI toolchain.

Reply:
moved to plugin-debug.